

Kconfig barfs if it doesn't have leading spaces in the help.

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/drivers/block/Kconfig |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)

diff -puN drivers/block/Kconfig~select-cpio_list-or-source-directory-for-initramfs-image-fix drivers/block/Kconfig
--- 25/drivers/block/Kconfig~select-cpio_list-or-source-directory-for-initramfs-image-fix	2004-09-26 17:23:25.536169432 -0700
+++ 25-akpm/drivers/block/Kconfig	2004-09-26 17:23:28.550711152 -0700
@@ -372,7 +372,7 @@ config INITRAMFS_SOURCE
 	    <maj>       major number of nod
 	    <min>       minor number of nod
 
-	If you are not sure, leave it blank.
+	  If you are not sure, leave it blank.
 
 config LBD
 	bool "Support for Large Block Devices"
_
